Added RSpec Tests (#1)
danielsokil/ruby_enumerables: README.md

> The goal of this project is to add new methods to the existing Enumerable module, a method that is identical to the next ones, but (obviously) does not use the original ones.

Reviewer: ```suggestion
The goal of this project is to make a working copy of some of the most popular methods from the `Enumerable` module of the Ruby programming language.

The list that follows shows what methods were copied.
```

Create #my_each
Create #my_each_with_index
Create #my_select
Create #my_all?
Create #my_any?
Create #my_none?
Create #my_count
Create #my_map

> Create #my_inject

Reviewer: Replace all this with a listing of the imitated methods, like follows.

- `#each` -> `#my_each`
- `#inject` -> `#my_inject`
...

## Install Dependencies

- `bundle install`

## Run Debugger

> - `rdebug-ide enumerables.rb`

Reviewer: ```suggestion
- `rdebug-ide enumerable.rb`
```

## Run Tests

- `rspec`

## Built With

- Ruby
- Rubocop

> - RSpec

Reviewer: Specify the versions, here is an example.

- Ruby (2.6.6)
- Rubocop (0.81)

To get a local copy up and running follow these simple example steps.

1. Open the terminal command line window
2. `git clone https://github.com/epinczinger/ruby_enumerables.git`

> 3. Execute `ruby enumerables.rb`

Reviewer: For this project, you are not creating an executable file, it doesn't make sense to issue `ruby enumerables.rb`, because this is a module, it doesn't interact with the user in any form.

```suggestion
3. run the tests to make sure all the methods do what they are supposed to do
```
